CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/RequestHandler/ReplicatorConfigurationRequestHandler.java

```diff
@@ -327,4 +362,182 @@ public boolean filtered(Document document, EnumSet<DocumentFlag> flags) {
     }
 }
 
+class LocalWinsCustomConflictResolver implements ConflictResolver {
+    private static final String TAG = "CCRREPLCONFIGHANDLER";
+    @Override
+    public Document resolve(Conflict conflict) {
+        Document localDoc = conflict.getLocalDocument();
```

Comment:
make sure doc is not null

Reply:
This resolver is only for the scenarios where we have both the docs preset. Null remote doc is covered by another resolver. Idea here is not to complicate the resolver implementation as we anyways do validation on python side for the expected outcome. 

Even in case of Null remote document, idea is to retain changes from local doc.

Reply:
actually the null object check is about to immediately exit the test if any of local or remote doc turns null. adding the null object check only helps for test framework fails the tests as expected, not by accident.

Reply:
Added null check